scripts/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": "..",
+    "outDir": "../build",
+    "lib": ["es2015"],
+    "target": "es5"
```

Comment:
nit: `tsconfig-google.json` has the same `lib` and `target` options. Unless you plan to override, you should drop them from config here.

Reply:
Done